models/capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
```

Comment:
- [ ] I get this error when I try to test your app. You have `nameable` here, but what you defined in BaseDecorator is `@nameable_obj`. You need to rename this to `@nameable_obj` or change one in the BaseDecorator to `@nameable`. Kindly ensure your methods work as expected.

  ![jdue](https://user-images.githubusercontent.com/36542986/210372729-c014a966-fc9e-4c8f-96f5-f0ab0ed9fe96.jpg)